receiver/zipkinreceiver/trace_receiver.go

```diff
@@ -239,17 +241,23 @@ func (zr *zipkinReceiver) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		receiverTagValue = zipkinV1TagValue
 	}
 	obsrecv.EndTracesOp(ctx, receiverTagValue, td.SpanCount(), consumerErr)
+	if consumerErr == nil {
+		// Send back the response "Accepted" as
+		// required at https://zipkin.io/zipkin-api/#/default/post_spans
+		w.WriteHeader(http.StatusAccepted)
+		return
+	}
 
-	if consumerErr != nil {
+	if consumererror.IsPermanent(consumerErr) {
+		// Permanent error.
```

Comment:
This comment isn't really helpful, as the conditional statement makes it clear that it's a permanent error. I'd say either explain why we're doing what we're doing here, or remove the comment.

Reply:
Removing it for now.